kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -50,25 +50,36 @@
       </icon-button>
     </assessment-wrapper>
 
-    <p class="page-description">{{ content.description }}</p>
+    <p>{{ content.description }}</p>
 
-    <download-button v-if="canDownload" :files="content.files" class="download-button-left-align"/>
 
     <div class="metadata">
       <p v-if="content.author">
-        <strong>{{ $tr('author') }}: </strong>{{ content.author }}
+        {{ $tr('author') }}: {{ content.author }}
       </p>
-      <p v-if="content.license">
-        <strong>{{ $tr('license') }}: </strong>{{ content.license }}
-      </p>
-      <p v-if="content.license_description">
-        <strong>{{ $tr('license') }}: </strong>{{ content.license_description }}
+
+      <p v-if="content.license" >
+        {{ $tr('license') }}: {{ content.license }}
+
+        <template v-if="content.license_description">
+          <span ref="licensetooltip">
+            <ui-icon icon="info_outline" :ariaLabel="$tr('licenseDescription')" class="license-tooltip"/>
+          </span>
+
+          <ui-popover trigger="licensetooltip" class="license-description">
+            {{ content.license_description }}
+          </ui-popover>
+        </template>
+
       </p>
+
       <p v-if="content.license_owner">
-        <strong>{{ $tr('copyrightHolder') }}: </strong>{{ content.license_owner }}
+        {{ $tr('copyrightHolder') }}: {{ content.license_owner }}
       </p>
     </div>
 
+    <download-button v-if="canDownload" :files="content.files" class="download-button"/>
```

Comment:
>  just not being used

Not sure what you mean with that.

The button was moved down based on the updated Learn mockups. 
https://projects.invisionapp.com/share/ZTA3FH2S2#/screens/232712226